libs/wingsdk/src/shared-aws/api.onrequest.inflight.ts

```diff
@@ -50,12 +51,31 @@ function mapApigatewayEventToCloudApiRequest(
     ...request.queryStringParameters,
     ...request.multiValueQueryStringParameters,
   };
+
+  const body = parseBody(request);
   return {
     path: request.path,
-    body: request.body ? JSON.parse(request.body) : "",
+    body: body,
     headers: request.headers as Record<string, string>,
     method: parseHttpMethod(request.httpMethod),
     query: sanitizeParamLikeObject(query),
     vars: sanitizeParamLikeObject(request.pathParameters ?? {}),
   };
 }
+
+/**
+ * Parse body to JSON or empty string
+ * @param body body
+ * @returns JSON body
+ */
+function parseBody(request: APIGatewayProxyEvent) {
+  if (!request.body) return "";
+
+  const contentType = Object.entries(request.headers)
+    .filter(([key, _]) => key.toLowerCase() === "content-type")
+    .map(([_, value]) => value?.toLowerCase())[0];
```

Comment:
filter isn't needed since we don't need to go through all the headers

```suggestion
  const contentType = Object.entries(request.headers)
    .find(([key, _]) => key.toLowerCase() === "content-type")?.[1];
```

Reply:
We should probably normalize headers at some point. The local sim probably already toLower()s them via express since headers are not supposed to be case sensitive.